myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -103,7 +103,7 @@ protected void configure() {
 
     MapBinder<String, TaskFactory> mapBinder = MapBinder.newMapBinder(binder(), String.class, TaskFactory.class);
     mapBinder.addBinding(NodeManagerConfiguration.NM_TASK_PREFIX).to(NMTaskFactoryImpl.class).in(Scopes.SINGLETON);
-    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations();
+    Map<String, ServiceConfiguration> auxServicesConfigs = cfg.getServiceConfigurations().orNull();
     if (auxServicesConfigs != null) {
```

Comment:
Seems like if we've got an Optional auxServices.isPresent would be appropriate to avoid null check.